### PR TITLE
just a little "ry" to our life :D

### PR DESCRIPTION
**Pull request by niilasoika**
change `Testausserveri &copy; 2021` to `Testausserveri ry &copy; 2021` cuz idk the formalities?

### CODE REVIEW

**Comment by niilasoika** [comment]
wait damn i should add the org id too

**Comment by niilasoika** [comment]
holdonholdon

**Comment by niilasoika** [comment]
now im done

**Comment by raikasdev** [comment]
👨🏼‍💻

File: index.html [hunk 1]
```diff
@@ -83,7 +83,7 @@ <h2>Projektit</h2>
             </p>
             <div class="footer">
                 <div class="wide">
-                    <a href="mailto:masterminds@testausserveri.fi">masterminds@testausserveri.fi</a><br>Testausserveri &copy; 2021
+                    <a href="mailto:masterminds@testausserveri.fi">masterminds@testausserveri.fi</a><br>Testausserveri ry &copy; 2021<br>Y-tunnus: 3235794-4
```
Comment (ahnl) [inline index.html:86]:
```suggestion
                    <a href="mailto:masterminds@testausserveri.fi">masterminds@testausserveri.fi</a><br>&copy; 2021 Testausserveri ry
```

**Review by ahnl — COMMENTED** [review]
Changing up some things.

File: index.html (hunk 1 above)
Comment (ahnl) [inline index.html:86]:
```suggestion
                    <a href="mailto:masterminds@testausserveri.fi">masterminds@testausserveri.fi</a><br>&copy; 2021 Testausserveri ry
```

**Comment by niilasoika** [comment]
i guess i did it

**Comment by ahnl** [comment]
Thank you for your contribution! Merging now.

**Pull request merged**